bin/Debugger.Sample/Debugger.Sample.cpp

```diff
@@ -468,11 +486,13 @@ int _cdecl wmain(int argc, wchar_t* argv[])
 
         // Similarly, create a single execution context. Note that we're putting it on the stack here,
         // so it will stay alive through the entire run.
-        IfFailError(CreateHostContext(runtime, arguments.scriptArgs, debugProtocolHandler.get(), &context), L"failed to create execution context.");
+        IfFailError(CreateHostContext(runtime, arguments.scriptArgs, &context), L"failed to create execution context.");
 
         // Now set the execution context as being the current one on this thread.
         IfFailError(JsSetCurrentContext(context), L"failed to set current context.");
 
+        IfFailError(RedirectConsoleToDebugger(debugProtocolHandler.get()), L"Failed to redirect to debugger's console");
```

Comment:
I think it would help to add a command line switch whether to redirect console. In my opinion, it can redirect by default, but would be good to be able to turn off so we can verify.